README.adoc

```diff
@@ -40,20 +40,19 @@ Open Liberty. If you run the application, Maven creates an Open Liberty server a
 the application on it. The application displays a simple web page with a link that, when 
 clicked, calls the servlet to return a simple response of `Hello! How are you today?`.
 
-One benefit of using Maven is that, once you have defined the details of the project and 
-any dependencies it has, Maven automatically handles downloading and installing any and 
-all of the dependencies.
+Using a build tool like Maven is beneficial as it handles the downloading and installation of all dependencies.
+Build tools also allow for defining automated tests that can be easily re-run on each build. 
+If the tests don't pass after changes are made, the build fails so you know you need to fix your code. 
 
-Another benefit of using Maven is that it can run automated tests on the application after 
-building it. You could, of course, test your application manually by starting a server 
-and pointing a web browser at the application URL. Automated tests are a much better 
-approach though because Maven can easily re-run the same tests each time it builds the 
-application. If the tests don't pass after you've made a change to the application, the 
-build fails so you know that you need to fix your code.
+Maven has some differences and benefits over another build tool called Gradle. 
```

Comment:
I think I would take out the first two sentences and reword this section like the following:
Choosing a build tool often comes down to personal preference, but there are some benefits to using Maven over Gradle. As a mature build tool, Maven has greater integrations with IDEs. Maven also has an extensive.....

Reply:
Suggested wording used with minor modifications to avoid a direct comparison to Gradle.